### PR TITLE
Added retrieval of data  

### PR DESCRIPTION
**Pull request by yasharth291**
Now the list of organization and list of users could be seen 

@DangaRanga  @palisadoes 

Capture2
Capture

### CODE REVIEW

File: src/screens/SuperAdminOrgPage/SuperAdminOrgPage.tsx [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+        image
+        _id
+        creator {
+          firstName
+          lastName
+        }
+        name
+      }
+    }
+  `;
```
Comment (DangaRanga) [inline src/screens/SuperAdminOrgPage/SuperAdminOrgPage.tsx:20]: Although this query is really short, please create a queries folder to house your grpahQL queries, and import them into your components.

So this query could be found in `queries/superAdmin.js`

**Review by DangaRanga — CHANGES_REQUESTED** [review]
Good job Yasharth, there are just a few minor changes to be made and your PR will be good to go.

Please see my review comments

File: src/screens/SuperAdminMemberPage/SuperAdminMemberPage.tsx [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+  const PEOPLE_LIST = gql`
+    query {
+      users {
+        firstName
+        lastName
+        image
+        _id
+      }
+    }
+  `;
```
[the remark this answers is not in the document]
Comment (yasharth291) [reply src/screens/SuperAdminMemberPage/SuperAdminMemberPage.tsx:17]: ok sir

File: src/screens/SuperAdminOrgPage/SuperAdminOrgPage.tsx (hunk 1 above)
Comment (yasharth291) [reply src/screens/SuperAdminOrgPage/SuperAdminOrgPage.tsx:20]: ok sir

File: src/components/UserCard/UserCard.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styles from './UserCard.module.css';
+
+interface Details {
+  key: any;
+  firstName: any;
+  lastName: any;
+  image: any;
+}
```
Comment (DangaRanga) [inline src/components/UserCard/UserCard.tsx:9]: Please use a descriptive name for your interfaces. `Details` does not indicate the purpose of this interface.

See the suggestion below
```suggestion
interface UserCardProps {
  key: any;
  firstName: any;
  lastName: any;
  image: any;
}
```

File: src/components/Queries/Queries.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```
Comment (DangaRanga) [inline src/components/Queries/Queries.tsx:1]: These aren't components, so they would not belong in the components directory.

Please place them in either `src/graphql`

or `src/queries`

File: src/components/OrganizationCard/OrganizationCard.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+import styles from './OrganizationCard.module.css';
+
+interface Details {
+  key: any;
+  image: string;
+  id: string;
+  name: string;
+  lastName: string;
+  firstName: string;
+}
```
Comment (DangaRanga) [inline src/components/OrganizationCard/OrganizationCard.tsx:12]: Please use a descriptive name for your interfaces. `Details` does not indicate the purpose of this interface.

See the suggestion below
```suggestion
interface OrganizationCardProps {
  key: any;
  image: string;
  id: string;
  name: string;
  lastName: string;
  firstName: string;
}
```

**Review by DangaRanga — CHANGES_REQUESTED** [review]

**Comment by yasharth291** [comment]
@DangaRanga

**Comment by DangaRanga** [comment]
> @DangaRanga

You still need to migrate your graphql queries to a `queries` folder/directory, and import your queries from there.

You also still need to move your `Queries.tsx` file out of your components directory, and change the file extension since it is not a component. Since the queries that you wrote so far are for the super admin, an appropriate file name would be `superAdmin.js` and it would be located in a `queries` directory.

So the file path would be

`src/queries/superAdmin.js`

I have attached a codesandbox.io link on how you could do this without breaking the existing functionality using the code you provided below:
https://codesandbox.io/s/bold-moon-4iq6q?file=/src/App.js

This is to allow for the concerns in your code to be easily separated. That being separating your query logic, from your component logic, and connecting them through imports.

**Comment by yasharth291** [comment]
@DangaRanga  please merge it sir

**Review by DangaRanga — APPROVED** [review]

**Pull request merged**